local-rest-scorer/src/main/java/ai/h2o/mojos/deploy/local/rest/config/ScorerConfiguration.java

```diff
@@ -10,7 +10,7 @@
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
-class ScorerConfiguration {
+public class ScorerConfiguration {
```

Comment:
This only seems to be needed to derive an empty class. Not sure we need that, but maybe so. Please double check :pray:.

Reply:
yea if we go with your above suggestion we don't need this either.